Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -4247,6 +4250,12 @@ void MainWindow::slotClientPreferences(bool /*checked =false */)
         ttSpeech = nullptr;
     }
 #endif
+#if QT_VERSION >= QT_VERSION_CHECK(6,8,0)
+    if (ttSettings->value(SETTINGS_TTS_ENGINE, SETTINGS_TTS_ENGINE_DEFAULT).toUInt() == TTSENGINE_QTANNOUNCEMENT && announcerObject == nullptr)
+        QObject* announcerObject = this;

```

Comment:
This looks incorrect

Reply:
Why incorrect?

Reply:
You create a local variable `announcerObject` pointing to `this`.

Reply:
Yes, because we need a QObject representing a object-class to use QAccessibleAnnouncementEvent, so main window is extern to UtilTTS. I didn't saw a better way, do you see one?

Reply:
but you just create an unused variable.

Isn't it the global variable `announcerObject` that you want to assign to this?

Reply:
Ohhh, yes. It should be better after 067b045